presto-main/src/main/java/com/facebook/presto/spiller/BinaryFileSpiller.java

```diff
@@ -53,19 +54,23 @@
     private final Closer closer = Closer.create();
     private final BlockEncodingSerde blockEncodingSerde;
     private final AtomicLong totalSpilledDataSize;
+    private final AbstractOperatorSpillContext operatorSpillContext;
 
     private int spillsCount;
     private final ListeningExecutorService executor;
+    private long spillerDataSize = 0L; // TODO: is there ever concurrent access to Spillers?
```

Comment:
no there is not - at least for now in aggregations. Spiller is owned by single `HashAggregationOperator`